sentry/src/main/java/io/sentry/Span.java

```diff
@@ -163,6 +188,30 @@ public void finish(final @Nullable SpanStatus status, final @Nullable SentryDate
 
     this.context.setStatus(status);
     this.timestamp = timestamp == null ? hub.getOptions().getDateProvider().now() : timestamp;
+    if (options.isTrimStart() || options.isTrimEnd()) {
```

Comment:
`m`: I'd also like to have it tested if possible

Reply:
There are some tests within `SentryTracer`, but let me add some to `Span` as well 👍